spec/logr/entry_spec.rb

```diff
@@ -0,0 +1,118 @@
+require 'spec_helper'
+
+describe Logr::Entry do
+  let(:logger) { double('logger') }
+  let(:entry) { Logr::Entry.new(logger) }
+
+  describe '#event' do
+    let(:entry_event) { entry.event('test_event', context_1_key: 'context_1_value') }
+
+    it 'should add the event with the correct name' do
+      expect(entry_event.instance_variable_get(:@event).name).to eq('test_event')
+    end
+
+    it 'should add the event with the correct context' do
+      expect(entry_event.instance_variable_get(:@event).context).to include(context_1_key: 'context_1_value')
```

Comment:
Yes we could allow that. Actually we should. As I think more about it, it makes sense that if we call it without any arguments it should just return the current value.